solana/solana-ibc/programs/solana-ibc/src/execution_context.rs

```diff
@@ -305,10 +305,13 @@ impl storage::IbcStorage<'_, '_> {
         client_id: &ibc::ClientId,
         state: AnyClientState,
     ) -> Result<(), ibc::ClientError> {
-        msg!("store_client_state({}, {:?})", client_id, state);
         let mut store = self.borrow_mut();
         let mut client = store.private.client_mut(client_id, true)?;
         client.client_state.set(&state)?;
+        msg!(
+            "This is updated client state {:?}",
+            client.client_state.as_bytes()
+        );
```

Comment:
Perhaps it’d be better to turn this into an Event? We could do the same with consensus state.

Reply:
Yes sounds good.